### ISSUE TITLE
loaderContext.options were removed from webpack 4.0.0-alpha.0 

### ISSUE THREAD

**Issue by sdoomz** [webpack-contrib/eslint-loader#204]
**eslint-loader** fails with error when running with webpack 4.0.0-alpha.0 https://github.com/webpack/webpack/issues/6064

output:
```
Module build failed: TypeError: Cannot read property 'eslint' of undefined
    at Object.module.exports (/Users/svyatoslav/Dev/wix-vod/wix-vod/node_modules/eslint-loader/index.js:148:18)
webpack: Failed to compile.
```

**Comment by dagstuan** [comment]
@MoOx: https://medium.com/webpack/webpack-4-migration-guide-for-plugins-loaders-20a79b927202

**Comment by brigand** [comment]
Simply switching `this.options.eslint` to `this.query` seems to work for me.

This was my config:

```js
{ loader: 'eslint-loader', options: { emitWarning: true } }
```

And in eslint-loader, `this.query` is `{ emitWarning: true }`.

**Comment by MoOx** [comment]
For anyone willing to work on this, see [this comment](https://github.com/webpack-contrib/eslint-loader/pull/210#issuecomment-365179162)

**Comment by MoOx** [comment]
Fixed in 2.0.0
